filters/include/pcl/filters/impl/plane_clipper3D.hpp

```diff
@@ -219,9 +219,9 @@ pcl::PlaneClipper3D<PointT>::clipPointCloud3D (const pcl::PointCloud<PointT>& cl
   }
   else
   {
-    for (std::vector<int>::const_iterator iIt = indices.begin (); iIt != indices.end (); ++iIt)
-      if (clipPoint3D (cloud_in[*iIt]))
-        clipped.push_back (*iIt);
+    for (const auto& iIt : indices)
```

Comment:
`It` is for iterator. A variable name change wouldn't be bad here (and other places)